packages/components/psammead-podcast-promo/src/components/card-description.jsx

```diff
@@ -0,0 +1,15 @@
+import styled from '@emotion/styled';
+import { getLongPrimer } from '@bbc/gel-foundations/typography';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { C_METAL } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING } from '@bbc/gel-foundations/spacings';
+
+const CardDescription = styled.p`
+  ${({ script }) => getLongPrimer(script)};
+  ${({ service }) => getSansRegular(service)};
+  max-width: 30rem;
```

Comment:
should all other px measurements use rem?

Reply:
I don't think so?  If people wanted to make everything bigger, they would use the page zoom feature.  REM should only be used in situations where we want things to change size based on the user's selected text size

We do use REM for other things too though - I don't really know what the BBC / Simorgh guidance is for when to use REM and when to use PX